### ISSUE TITLE
Missing license

### ISSUE THREAD

**Issue by mk3z** [simtrami/posy-improved-cursor-linux#9]
A license would make distributing this cursor theme easier.

**Comment by simtrami** [comment]
Indeed, but I do not know how I can license the cursor files as I am not their original author and Posy did not license its own work. Any idea?

**Comment by mk3z** [comment]
I can ask Posy if he would be willing to add a license to the cursors.

**Comment by michaelBelsanti** [comment]
He has added an attribution-noncommercial license
https://michieldb.nl/other/cursors/
https://creativecommons.org/licenses/by-nc/4.0/deed.en

**Comment by simtrami** [comment]
I'm adding it right now, thank you very much for looking into this!

**Comment by simtrami** [comment]
Closed by ffef15950490af620fd0f70cc98847880f63d96e
Thanks everyone!
